backend/pong/match/tests/test_message_exceptions.py

```
@@ -0,0 +1,82 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from match.consumers import MultiEventConsumer
+from channels.layers import get_channel_layer
```

Ruff (0.8.2)

`channels.layers.get_channel_layer` imported but unused

Remove unused import: `channels.layers.get_channel_layer`

(F401)